app/src/main/java/org/torproject/android/ui/onboarding/CustomSlideBigText.java

```diff
@@ -19,6 +22,8 @@
     private String mButtonText;
     private String mSubTitle;
     private View.OnClickListener mButtonListener;
+    private TextView tv, title;
```

Comment:
Perhaps `tv` should be renamed to something more specific

Reply:
I let the default names stay. Yeah, I should probably.